Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM quay.io/ukhomeofficedigital/alpine:v3.13
+
+ENV USER user-hocs-bulk-case-action
+ENV USER_ID 1000
+ENV GROUP group-hocs-bulk-case-action
+ENV NAME hocs-bulk-case-action
+ENV JAR_PATH build/libs
+
+ENV workflow-address http://host.docker.internal:8091
+ENV x-auth-groups /COMP_CASEADMIN_bdhvyvg
+ENV x-auth-userId 968672ca-9fc1-491c-b2b9-d13c58b12d7a
+ENV gap-between-updates 5000
+ENV file-path ./test.csv
```

Comment:
Unsure on this, are we saying we need to check in a file for this when we deploy? This is set to start processing on start up, but the file wouldn't be there if. I wouldn't be comfortable putting production data into the repo.

Reply:
I'm not sure on how we should feed the file in at the moment... this needs thought

Reply:
The initial idea I had was we expose a endpoint that supports a MultipartFile. Maybe something to investigate?

Reply:
Worth having a call about this if a solution hasn't already been thought out?

Reply:
We got permission last week to store the CSV in github so I think that'll be the route